test/windows/interactive-win11-validate.ps1

```
@@ -0,0 +1,177 @@
+    }
+}
+
+function Get-HarnessArguments {
+    param(
+        [Parameter(Mandatory)] [string] $ScriptName,
+        [int] $TimeoutSeconds = 0,
+        [switch] $IncludeResetState
+    )
+
+    $scriptPath = Join-Path $PSScriptRoot $ScriptName
+    $args = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)